sheep/src/lib.rs

```diff
@@ -53,20 +71,28 @@ pub fn pack<P: Packer>(
 
     packer_result
         .into_iter()
-        .map(|sheet| {
+        .map(|mut sheet| {
             let mut buffer = create_pixel_buffer(sheet.dimensions, stride);
-            sprites
-                .iter()
-                .filter_map(|sprite| {
-                    sheet
-                        .anchors
+            let mut additional_anchors = Vec::<SpriteAnchor>::new();
+            for anchor in &sheet.anchors {
+                write_sprite(
+                    &mut buffer,
+                    sheet.dimensions,
+                    stride,
+                    &sprites[anchor.id],
+                    &anchor,
+                );
+                additional_anchors.extend(
```

Comment:
Changing the name here to `aliased_anchors` would make it clearer what's happening, imo.